packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -226,6 +253,90 @@ contract VerifyOPCM is Script {
         return refs;
     }
 
+    /// @notice Verifies that all OPCM component contracts have the same contractsContainer address.
+    /// @param _propRefs Array of property references containing component addresses.
+    function _verifyContractsContainerConsistency(OpcmContractRef[] memory _propRefs) internal view {
+        // Filter components that have contractsContainer() function
+        address[] memory componentAddresses = new address[](_propRefs.length);
+        string[] memory componentNames = new string[](_propRefs.length);
+        uint256 componentCount = 0;
+
+        for (uint256 i = 0; i < _propRefs.length; i++) {
+            // Only check components that have contractsContainer() function
+            string memory field = _propRefs[i].field;
+            if (
+                bytes(field).length > 0
+                    && (
+                        LibString.eq(field, "opcmGameTypeAdder") || LibString.eq(field, "opcmUpgrader")
+                            || LibString.eq(field, "opcmDeployer") || LibString.eq(field, "opcmInteropMigrator")
```

Comment:
let's modify this so that it's true if the first 4 letters match `"opcm"`, that way newly added components will be covered here.